src/dsl_vlplot_function/dsl_vlplot_function.jl

```diff
@@ -0,0 +1,223 @@
+struct VLFrag
+    positional::Vector{Any}
+    named::Dict{String,Any}
+end
+
+function vlfrag(args...; kwargs...)
```

Comment:
Maybe only provide 0-arg and 1-arg signatures (as it seems that there are not multi-arg support ATM)?  Or maybe even just only 1-arg, as you can use a named tuple instead of 0-arg `vlfrag`.

I think it is much nicer to error out at the call site rather than doing so later.

Reply:
+1, https://github.com/queryverse/VegaLite.jl/pull/233